mlir/lib/Dialect/Tosa/Transforms/TosaValidation.cpp

```diff
@@ -695,6 +773,9 @@ LogicalResult TosaValidation::applyErrorIfCheck(Operation *op) {
 }
 
 bool TosaValidation::isValidElementType(Type type) {
+  if (auto quantType = llvm::dyn_cast<mlir::quant::QuantizedType>(type))
```

Comment:
Should quantized types have already been stripped at this point?

Reply:
Yes, you are right. Quantized types will be stripped by the pass preceding this validation. Can be removed.

Reply:
Sorry, I thought it was but it wasn't. The quantized types are stripped by framework, e.g. Tensorflow, hence it is not guarantee quantized types have already been stripped at this point. We already have some lit tests expecting operating quantized types, e.g. ops.mlir.

Reply:
Hmm, my understanding was that the validation pass can be used as the final check before converting to other formats such as flatbuffer which do not support quantized types. Did we rely on this behaviour before now? I think some quantized test cases were added in this PR, but since they're checking the level, can they be swapped out with more basic types?

> We already have some lit tests expecting operating quantized types, e.g. ops.mlir

In ops.mlir, I think quantized types are okay since we're not running the validation pass

Reply:
make sense. I agree your point that the validation pass is usually used at the end of compilation as the final check. Let me swap out the quantized type with basic types from the tests.